website/sidebars.json

```diff
@@ -336,6 +336,13 @@
         "tutorials/examples/update-contract-migrate-state"
       ]
     },
+    "tutorials/examples/hello-near",
+    "tutorials/examples/count-near",
+    "tutorials/examples/guest-book",
+    "tutorials/examples/donation",
+    "tutorials/examples/escrow",
+    "tutorials/examples/xcc",
+    "tutorials/examples/coin-flip",
```

Comment:
these examples are already listed under the "Basic Examples" sidebar group.

I'd add the escrow example there, but if you think it should remain outside the group, ping Guillermo just to be sure he's ok with it.

Reply:
Thanks for pointing this out! I must have missed it while merging master. I've added `escrow` to the "Advanced Examples" area